deploy/neon/values.yaml

```diff
@@ -0,0 +1,129 @@
+version: latest
+
+image:
+  # -- Neondatabase image repository
+  repository: perconalab/neon
+
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+  # -- Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
```

Comment:
Better to use a specified version, the CD may be unsuitable for the latest image.